components/cqrs/name.go

```diff
@@ -0,0 +1,42 @@
+package cqrs
+
+import (
+	"fmt"
+	"strings"
+)
+
+// ObjectName name returns object name in format [package].[type name].
+// It ignores if the value is a pointer or not.
+func ObjectName(v interface{}) string {
+	s := fmt.Sprintf("%T", v)
+	s = strings.TrimLeft(s, "*")
+
+	return s
+}
+
+// StructName name returns struct name in format [type name].
+// It ignores if the value is a pointer or not.
+func StructName(v interface{}) string {
```

Comment:
maybe ObjectNameWithoutPackage? it is pretty long, but explicit :)

Reply:
I was rather thinking about renaming `ObjectName` to `FullyQualifiedStructName` and leave this as is, but `ObjectNameWithoutPackage` works as well.

Reply:
ok, like I like the idea of FullyQualifiedStructName :)